ocs_ci/helpers/e2e_helpers.py

```
@@ -113,6 +153,1046 @@ def get_all_combinations_map(providers, bucket_types):
+                obj_key,
+                versions[0]["VersionId"],
+            )
+            return versions[0]
+
+        logger.warning(
+            "Waiting for non-current versions to expire for %s/%s (%s version(s) remain)",
+            bucket_name,
+            obj_key,
+            len(versions),
+        )
+    else:
```

Ruff (0.15.17)

[warning] `else` clause on loop without a `break` statement; remove the `else` and dedent its contents

Remove `else`

(PLW0120)